examples/rhino-sdk/federated_join/Federated Join Tutorial.ipynb

```diff
@@ -0,0 +1,585 @@
+    "    join_field={\"data_column\": \"UID\", \"filter_column\": \"Age\", \"filter_value\": 35, \"filter_type\": \">\"},\n",
+    "    group_by={\"groupings\": [\"Gender\"]},\n",
+    ")\n",
+    "\n",
+    "joined_results = session.project.joined_cohort_metric(\n",
+    "    filter_cohorts=[filter_cohort_at_lahey.uid],\n",
+    "    data_cohorts=[first_data_cohort_at_mgh.uid],\n",
+    "    configuration=configuration\n",
+    ")\n",
+    "\n",
+    "print(joined_results.output)\n",
+    "\n",
+    "c = first_data_df[\"Gender\"] == \"f\"\n",
+    "female_centralized_result = np.mean(first_data_df.SpO2[a&c])\n",
+    "print(female_centralized_result == joined_results.output['f']['mean'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 100,
+   "id": "ede912ee",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Filtering on multiple data columns\n",
+      "{'mean': 1.0492307692307692}\n",
+      "1.0492307692307692\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Filtering on multiple data columns\")\n",
+    "\n",
+    "configuration = Mean(\n",
+    "    variable=\"SpO2\",\n",
+    "    join_field=\"UID\",\n",
+    "    data_filters=[\n",
+    "        {\n",
+    "            \"filter_column\": \"Age\",\n",
+    "            \"filter_value\": 35,\n",
+    "            \"filter_type\": \">\",\n",
+    "            # For intersection mode, if unspecified apples to both cohorts. \n",
+    "        },\n",
+    "        {\n",
+    "            \"filter_column\": \"BloodType\",\n",
+    "            \"filter_value\": \"a\",\n",
+    "            \"filter_type\": \"=\",\n",
+    "            \"filter_cohort\": filter_cohort_at_lahey.uid,\n",
+    "        },\n",
+    "        {\n",
+    "            \"filter_column\": \"Gender\",\n",
+    "            \"filter_value\": \"m\",\n",
+    "            \"filter_type\": \"=\",\n",
+    "            \"filter_cohort\": first_data_cohort_at_mgh.uid,\n",
+    "        }\n",
+    "    ],\n",
+    ")\n",
+    "\n",
+    "joined_results = session.project.joined_cohort_metric(\n",
+    "    filter_cohorts=[filter_cohort_at_lahey.uid],\n",
+    "    data_cohorts=[first_data_cohort_at_mgh.uid],\n",
+    "    configuration=configuration\n",
+    ")\n",
+    "print(joined_results.output)\n",
+    "federated_result = joined_results.output['mean']\n",
+    "\n",
+    "a = filter_df[\"Age\"] > 35\n",
+    "b = filter_df[\"BloodType\"] == \"a\"\n",
+    "valid_uids = first_data_df.UID.isin(filter_df[a&b].UID)\n",
+    "c = first_data_df[\"Age\"] > 35\n",
+    "d = first_data_df[\"Gender\"] == \"m\"\n",
+    "centralized_result = np.mean(first_data_df.SpO2[valid_uids&c&d])\n",
+    "print(centralized_result)\n",
+    "print(centralized_result == federated_result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa2bd257",
+   "metadata": {},
+   "source": [
+    "# Union Mode\n",
```

Comment:
Give me a dscription of what Union Mode is. If there is supporting documentation either in the SDK or on ZenDesk maybe link there